tools/btfgen.sh

```diff
@@ -18,20 +18,24 @@ if [ ! -d "${INSPEKTOR_GADGET}" ]; then
     exit 1
 fi
 
-${BTFHUB}/tools/btfgen.sh -a ${ARCH}                                                    \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/audit-seccomp/tracer/bpf/audit-seccomp.o         \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/bindsnoop/tracer/core/bindsnoop_bpfel.o          \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/biotop/tracer/biotop_bpfel.o                     \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/execsnoop/tracer/core/execsnoop_bpfel.o          \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/filetop/tracer/filetop_bpfel.o                   \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/fsslower/tracer/core/fsslower_bpfel.o            \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/mountsnoop/tracer/core/mountsnoop_bpfel.o        \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/oomkill/tracer/oomkill_bpfel.o                   \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/opensnoop/tracer/core/opensnoop_bpfel.o          \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/seccomp/tracer/bpf/seccomp.o                     \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/sigsnoop/tracer/core/sigsnoop_bpfel.o            \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/tcpconnect/tracer/core/tcpconnect_bpfel.o        \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/tcptop/tracer/tcptop_bpfel.o                     \
+${BTFHUB}/tools/btfgen.sh -a ${ARCH}                                                                \
```

Comment:
I know it was not done before, but could you add a comment before saying that network eBPF programs like dns and snisnoop are not included here because they don't need BTF?